api/capi/include/nnstreamer.h

```diff
@@ -238,6 +244,16 @@ typedef void (*ml_pipeline_sink_cb) (const ml_tensors_data_h data, const ml_tens
  */
 typedef void (*ml_pipeline_state_cb) (ml_pipeline_state_e state, void *user_data);
 
+/**
+ * @brief Callback for custom condition of tensor_if
+ * @since_tizen 6.5
+ * @param[out] info The handle of tensors information (cardinality, dimension, and type of given tensor/tensors).
+ * @param[out] data The handle of the tensor output (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
+ * @param[out] user_data User application's private data.
+ * @param[out] result Result of the user defined condition
```

Comment:
how about returning int value as result? not result ptr.
int if_custom_cb (info, data, user_data)

Reply:
How about returning the error code?

Reply:
Are you able to handle error returns from this callback?

Reply:
The callback function defined by tensor-if returns TRUE if no error. So, I agree with @gichan-jang 's intention.

Reply:
```Result of the user defined condition``` 
==>
```Result of the user-defined condition. 0 refers to FALSE and a non-zero value refers to TRUE.```

Reply:
Thanks. I will update.